### DISCUSSION TITLE
[LAMMPS] - Redundant output printing

### DISCUSSION THREAD

**Discussion by AnuragKr** [deepmodeling/deepmd-kit#2042] [Q&A] [answered]
Hello,

I ran the LAMMPS simulation using following command--
```bash
mpirun -np 2 lmp -in in.lammps
```

System Configuration - NVIDIA DGX V100
Output --
![Screenshot 2022-11-01 at 4 31 21 PM](https://user-images.githubusercontent.com/10074710/199226038-8b01699e-2b47-4240-a6e7-03ba841f15e7.png)

As we can see above that every step is printed twice. Is it normal or is something wrong? I think it is running two different lammps simulations on two nodes. GPU is also not getting utilized uniformly.
GPU Utilization--
![Screenshot 2022-11-01 at 4 29 35 PM](https://user-images.githubusercontent.com/10074710/199227741-81348e96-802e-4a8e-a2a0-be147b42f16d.png)

I ran same script on NVIDIA GTX 2080 Ti but here it printed once only and it ran faster than V100.

Lammps Code--
```lammps
#Starting
units metal
atom_style atomic
boundary p p p
read_data data/ice_pimd.dat
pair_style deepmd ../../ice_pimd_single_gpu_graph-compress.pb
pair_coeff * *

timestep 0.00005

group Oatoms type 1
group Hatoms type 2

replicate 2 2 2
fix 1 all npt temp 300 300 0.0005 iso 1.0 1.0 0.005

thermo_style custom step temp ke pe etotal press vol
thermo 100
thermo_modify format line "%d %.6e %.6e %.6e %.6e %.6e %.6e %.6e %.6e %.6e"

dump my_dump all custom 100 result/ice_pimd_10000_fs.xyz id type x y z vx vy vz
dump_modify my_dump sort id
dump my_dump2 all atom 100 result/ice_pimd_10000.lammps-dump-text

run 10000
```

Thanks

**Comment by njzjz** [comment, marked as answer]
See #1978

**Reply by AnuragKr** [reply]
Thanks @njzjz.

**Comment by AnuragKr** [comment]
Mine Solution -- Outside the conda environment, I installed OpenMPI-4.1.4 earlier so I disabled it, and then lammps started working as expected in the conda deepmd environment. As the MPI version is 3.3.2 in the deepmd environment, it might be creating conflict with the locally installed MPI.
